docs/standardisation-scope.adoc

```diff
@@ -0,0 +1,199 @@
+= Standardisation scope of Cypher
+
+One of the challenges faced by this effort to standardise Cypher has been to choose the set of language features to standardise.
+Cypher is not a new language, but has several years of history, which includes some historical artifacts left in the feature set.
```

Comment:
"Cypher is not a new language, but has several years of history, which includes some historical artifacts left in the feature set."
-->
"As Cypher has several years of history, some historical artifacts are present in the feature set."